generate_profiles/BuildXmlFiles.groovy

```diff
@@ -1,11 +1,9 @@
-import groovy.text.SimpleTemplateEngine
 import groovy.xml.MarkupBuilder
-import groovy.util.Node
 
 @Grapes([
     @Grab(group='com.google.code.findbugs'  , module='findbugs', version='3.0.1'),
-    @Grab(group='com.mebigfatguy.fb-contrib', module='fb-contrib', version='6.6.1'),
-    @Grab(group='com.h3xstream.findsecbugs' , module='findsecbugs-plugin', version='1.4.6')]
+    @Grab(group='com.mebigfatguy.fb-contrib', module='fb-contrib', version='6.8.0'),
```

Comment:
Unfortunately I was not able to extract version numbers to constants as groovy does not like it